geo.go

```diff
@@ -50,6 +51,18 @@ func geoObjectResult(o *objectResult) (err error) {
 	if countryName == "" {
 		countryName = "Unknown"
 	}
+
+	// Check if the ip is part of our custom overrides
+	for _, override := range cfg.overrides{
+		_, subnet, _ := net.ParseCIDR(override.CIDR)
+		if subnet.Contains(ip) {
+			cityName = override.City
+			countryName = override.Country
+			o.Latitude = override.Latitude
+			o.Longitude = override.Longitude
```

Comment:
Should we break out of the loop at this point if we've found a match?

Reply:
Good idea, added in https://github.com/ameihm0912/geomodel/pull/10/commits/03a52ec19ccfe82ab063817955f81fe3b83223ec